src/main.ts

```diff
@@ -284,6 +286,15 @@ export function prettifyExecutionGraphResult(executionGraphResult: Object): void
           ansi.red(task["tests"]["failed"])
         )} ${ansi.bold(ansi.red("failed"))}`
       )
+      core.summary
+      .addTable([task["action_id"], "action:", `${task["passed"] ? ("passed") : ("failed")}`])
```

Comment:
Looks like a missing bracket is breaking the summary execution (?):

```suggestion
      .addTable([[task["action_id"], "action:", `${task["passed"] ? ("passed") : ("failed")}`]])
```

I guessed it bc you're using double brackets on the vulns' table. And because the tests table wasn't printed in the summary, neither the latest raw sentence 🤔